apps/nuxt3-ssr/components/SearchResultsCount.vue

```diff
@@ -0,0 +1,25 @@
+<template>
```

Comment:
(almost ) all vue3 / nuxt / catalogue SFC have the setup function ( script tags) before the template tag , could you flip these to be consistent with the other files